<!-- place this in covins_frontend/launch/ -->
<?xml version="1.0"?>
<launch>

<arg name="ag_n" default="0" />
<arg name="cam_topic" default="hololens/vlc_image" />
<arg name="odom_topic" default="hololens/odom" />
<arg name="config_file" default="$(find covins_frontend)/config/covins/hlvlc_rf.yaml" />

<node pkg="covins_frontend" type="covins_frontend_node" name="frontend_ag$(arg ag_n)" output="screen">
    <param name="config_file" type="str" value="$(arg config_file)" />
    <remap from="/camera/image_raw" to="$(arg cam_topic)"/>
    <remap from="/cam_odom" to="$(arg odom_topic)"/>

</node>

</launch>